mmcv/datasets/pipelines/transforms.py

```diff
@@ -45,9 +40,12 @@ def __init__(self, mean: Sequence, std: Sequence, to_rgb: bool = True):
     def __call__(self, results: dict) -> dict:
         """Call function to normalize images.
 
+        The required key of `results` is `img_fields`, which is a list of
```

Comment:
Single backquote show as italic, if you want to show as code, need to use double backquotes. Refers to [the docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup)

Reply:
got it